src/components/widgets/SymbolSelector.js

```diff
@@ -81,7 +81,10 @@ export default class SymbolSelector extends Component {
         <div
           className="symbol-selector__item"
           key={value}
-          onClick={() => this.props.onChange(value)}
+          onClick={() => {
+            this.props.onChange(value);
+            this.togglePanel();
```

Comment:
ok, although eventually, for consistency we'll probably need to do the same as with the color picker, where it appeared in line.

Reply:
Yep, raised an issue for that! #706

Reply:
thx